app/views/components/summaryCardLSP.scala.html

```diff
@@ -105,7 +105,7 @@ <h4 class="app-summary-card__title">
                                 </a>
                         } else {
                                 <a class="govuk-link" href={controllers.routes.IndexController.redirectToAppeals(summaryCard.penaltyId, false, false, false).url}>
-                                {messages("summaryCard.footerLinkText", summaryCard.penaltyPoint)} <span class="govuk-visually-hidden">{visuallyHiddenText.getOrElse("")}</span>
+                                {messages("summaryCard.footerLinkText", summaryCard.penaltyPoint)}
```

Comment:
We still need the visually hidden text, however we shouldn't show it when it's empty
Maybe mapping over `visuallyHiddenText` may work - something like:
`visuallyHiddenText.map {text => <span class="govuk-visually-hidden">@text</span>}` after the message

Reply:
Would agree here, but code uses else statements and the removal was required for appeal penalty point else statement